src/main/scala/dxWDL/dx/DxWorkflow.scala

```diff
@@ -172,6 +172,47 @@ case class DxWorkflow(id: String, project: Option[DxProject]) extends DxExecutab
         throw new Exception(s"malformed json response ${other}")
     }
   }
+
+  def setDetails(jsValue: JsValue): Unit = {
+    DXAPI.workflowSetDetails(
+      id,
+      DxUtils.jsonNodeOfJsValue(jsValue),
+      classOf[JsonNode],
+      DxUtils.dxEnv
+    )
+  }
+
+  def updateDetails(jsObject: JsObject): Unit = {
```

Comment:
This is going to cost two additional API calls. Can we fold these changes into the `DXAPI.workflowNew` call ?